### PR TITLE
Role for Doplarr

### PR DESCRIPTION
**Pull request by grostim**
This is a PR for #18 

It is based upon docker image from Doplarr dev: https://github.com/kiranshila/Doplarr

This base version has been tested successfully working together with Overseer.
At this stage, i have not tried working directly with Radarr/Sonarr but this should work easily by customizing doplarr_docker_envs_custom

### CODE REVIEW

**Comment by saltydk** [comment]
fail conditions need to be skipped under CI.

File: roles/doplarr/tasks/main.yml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+#########################################################################
+#                   GNU General Public License v3.0                     #
+#########################################################################
+---
+- name: Fail when default doplarr.discord_token is specified
+  fail: msg="You must specify a Discord bot token in the settings under doplarr.discord_token ."
+  when: doplarr.discord_token == 'your_discord_bot_token'
+
+- name: Fail when default doplarr.overseerr_api is specified
+  fail: msg="You must specify ayour Overeerr API key in the settings under doplarr.overseerr_api ."
```
Comment (saltydk) [inline roles/doplarr/tasks/main.yml:15]: spelling? :P

**Comment by grostim** [comment]
> fail conditions need to be skipped under CI.

How should i do that ?

**Comment by owine** [comment]
> > fail conditions need to be skipped under CI.
> 
> How should i do that ?

`when: not continuous_integration`

**Pull request merged**